### PR TITLE
Add endpoint /views/internal/server/now

### PR DESCRIPTION
**Pull request by leszko**
Related to https://www.notion.so/livepeer/Cap-50k-Fishtank-viewers-a828ef8d037a4f27b9a6769d91c50b23

### CODE REVIEW

File: api/handler.go [hunk 1]
```diff
@@ -133,6 +133,10 @@ func (h *apiHandler) viewershipHandler() chi.Router {
 		With(h.cache(false)).
 		MethodFunc("GET", fmt.Sprintf(`/{%s}/total`, assetIDParam), h.getTotalViews)
 
+	// realtime viewership server side (internal-only)
+	h.withMetrics(router, "query_realtime_server_viewership").
+		MethodFunc("GET", "/internal/server/now", h.queryRealtimeServerViewership())
```
Comment (thomshutt) [inline api/handler.go:138]: Should we add something into the path to indicate viewership? e.g (bit verbose though) `/internal/server/viewership/now`

**Review by thomshutt — APPROVED** [review]

File: api/handler.go (hunk 1 above)
Comment (leszko) [reply api/handler.go:138]: the viewership handler is already defined as `/views`, so the full path will be `/data/views/internal/server/now`.

Comment (thomshutt) [reply api/handler.go:138]: Ah gotcha, I'd missed that - LGTM then

File: api/handler.go [hunk 2]
```diff
@@ -558,6 +562,29 @@ func (h *apiHandler) getTotalViews(rw http.ResponseWriter, r *http.Request) {
 	respondJson(rw, http.StatusOK, totalViews)
 }
 
+func (h *apiHandler) queryRealtimeServerViewership() http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		userId := r.URL.Query().Get("userId")
```
Comment (victorges) [inline api/handler.go:567]: Could we allow querying by `playbackId` here? Would make it super easy to migrate the existing per-stream `rate_limit` to this if so

**Review by victorges — APPROVED** [review]
LGTM

File: api/handler.go (hunk 2 above)
Comment (leszko) [reply api/handler.go:567]: Yes, we can, but let's leave it as a separate task. I've created a Linear ticket about it: https://linear.app/livepeer/issue/ENG-2120/change-rate-limite-to-viewer-limit

**Pull request merged**